src/app/app.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { TestBed, async } from '@angular/core/testing';
```

Comment:
delete this file until you start writing tests and running them. atm its just polluting  your pr